{
 "PRed": [
  {
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.11"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 6,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.12"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 7,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "CompilerRebuild",
    "migrator_version": 1,
    "name": "Python 3.7, GCC 7, R 3.5.1, openBLAS 0.3.2"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 8,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.13"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 9,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.14"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 12,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.15"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 13,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Rebuild",
    "migrator_object_version": 3,
    "migrator_version": 0,
    "name": "libprotobuf-3.7"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 14,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.16"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 16,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Rebuild",
    "migrator_version": 0,
    "name": "libprotobuf-3.8"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 17,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.17"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 18,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.18"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 19,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "python38"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 20,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "protobuf310"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 21,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "protobuf311"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 23,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.19"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 24,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "pypy"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 26,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.20"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 27,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "libprotobuf312"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 30,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.21"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 34,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "libprotobuf313"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 35,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 2,
    "migrator_version": 0,
    "name": "python39"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 36,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.22"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 37,
    "state": "closed"
   },
   "data": {
    "bot_rerun": 1611222152.835995,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.23"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 38,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "libprotobuf314"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 39,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "pypy37"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 40,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.23"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 41,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "libprotobuf315"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 42,
    "state": "closed"
   },
   "data": {
    "bot_rerun": 1620488219.592155,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.24"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 43,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "libprotobuf316"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 44,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.24"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 45,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.25"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 46,
    "state": "closed"
   },
   "data": {
    "bot_rerun": 1634255448.9950218,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.26"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 47,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "libprotobuf318"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 51,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "pypy37-windows"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 54,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.28"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 57,
    "state": "closed"
   },
   "data": {
    "bot_rerun": 1644790614.9260647,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "libprotobuf319"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 58,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "python310"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 59,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "OSXArm",
    "migrator_version": 1,
    "name": "arm osx addition"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 60,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "libprotobuf319"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 61,
    "state": "closed"
   },
   "data": {
    "bot_rerun": 1651846945.8700733,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "pypy38"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 62,
    "state": "closed"
   },
   "data": {
    "bot_rerun": 1651846945.8702645,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "libprotobuf320"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 63,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.29"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 64,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "libprotobuf320"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 65,
    "state": "closed"
   },
   "data": {
    "bot_rerun": 1653887612.0689442,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "pypy38"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 67,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "pypy38"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 68,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.30"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 69,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "libprotobuf321"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 70,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.31"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "number": 73,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "python311"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "labels": [],
    "number": 78,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "8.0.32"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  }
 ],
 "bad": {
  "exception": "<class 'requests.exceptions.ConnectTimeout'>: A connection-level exception occurred: HTTPSConnectionPool(host='api.github.com', port=443): Max retries exceeded with url: /repos/conda-forge/mysql-connector-python-feedstock (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fbfcb4f6460>, 'Connection to api.github.com timed out. (connect timeout=4)'))",
  "traceback": [
   "Traceback (most recent call last):",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/urllib3/connection.py\", line 174, in _new_conn",
   "    conn = connection.create_connection(",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/urllib3/util/connection.py\", line 95, in create_connection",
   "    raise err",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/urllib3/util/connection.py\", line 85, in create_connection",
   "    sock.connect(sa)",
   "socket.timeout: timed out",
   "",
   "During handling of the above exception, another exception occurred:",
   "",
   "Traceback (most recent call last):",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/urllib3/connectionpool.py\", line 703, in urlopen",
   "    httplib_response = self._make_request(",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/urllib3/connectionpool.py\", line 386, in _make_request",
   "    self._validate_conn(conn)",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/urllib3/connectionpool.py\", line 1042, in _validate_conn",
   "    conn.connect()",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/urllib3/connection.py\", line 363, in connect",
   "    self.sock = conn = self._new_conn()",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/urllib3/connection.py\", line 179, in _new_conn",
   "    raise ConnectTimeoutError(",
   "urllib3.exceptions.ConnectTimeoutError: (<urllib3.connection.HTTPSConnection object at 0x7fbfcb4f6460>, 'Connection to api.github.com timed out. (connect timeout=4)')",
   "",
   "During handling of the above exception, another exception occurred:",
   "",
   "Traceback (most recent call last):",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/requests/adapters.py\", line 486, in send",
   "    resp = conn.urlopen(",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/urllib3/connectionpool.py\", line 787, in urlopen",
   "    retries = retries.increment(",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/urllib3/util/retry.py\", line 592, in increment",
   "    raise MaxRetryError(_pool, url, error or ResponseError(cause))",
   "urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.github.com', port=443): Max retries exceeded with url: /repos/conda-forge/mysql-connector-python-feedstock (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fbfcb4f6460>, 'Connection to api.github.com timed out. (connect timeout=4)'))",
   "",
   "During handling of the above exception, another exception occurred:",
   "",
   "Traceback (most recent call last):",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/github3/models.py\", line 201, in _request",
   "    return request_method(*args, **kwargs)",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/requests/sessions.py\", line 602, in get",
   "    return self.request(\"GET\", url, **kwargs)",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/github3/session.py\", line 171, in request",
   "    response = super().request(*args, **kwargs)",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/requests/sessions.py\", line 589, in request",
   "    resp = self.send(prep, **send_kwargs)",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/requests/sessions.py\", line 703, in send",
   "    r = adapter.send(request, **kwargs)",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/requests/adapters.py\", line 507, in send",
   "    raise ConnectTimeout(e, request=request)",
   "requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.github.com', port=443): Max retries exceeded with url: /repos/conda-forge/mysql-connector-python-feedstock (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fbfcb4f6460>, 'Connection to api.github.com timed out. (connect timeout=4)'))",
   "",
   "During handling of the above exception, another exception occurred:",
   "",
   "Traceback (most recent call last):",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/auto_tick.py\", line 1187, in _run_migrator",
   "    migrator_uid, pr_json = run(",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/auto_tick.py\", line 217, in run",
   "    feedstock_dir, repo = get_repo(",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/git_utils.py\", line 286, in get_repo",
   "    repo = gh.repository(\"conda-forge\", feedstock_reponame)",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/github3/github.py\", line 2103, in repository",
   "    json = self._json(self._get(url), 200)",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/github3/models.py\", line 216, in _get",
   "    return self._request(\"get\", url, **kwargs)",
   "  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/github3/models.py\", line 206, in _request",
   "    raise exceptions.ConnectionError(exc)",
   "github3.exceptions.ConnectionError: <class 'requests.exceptions.ConnectTimeout'>: A connection-level exception occurred: HTTPSConnectionPool(host='api.github.com', port=443): Max retries exceeded with url: /repos/conda-forge/mysql-connector-python-feedstock (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fbfcb4f6460>, 'Connection to api.github.com timed out. (connect timeout=4)'))",
   ""
  ]
 },
 "pinning_version": "2023.03.04.06.16.51",
 "pre_pr_migrator_attempts": {
  "protobuf423": 49,
  "pypy37": 1
 },
 "pre_pr_migrator_status": {
  "protobuf423": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/5414767593\">bot CI job</a>): main: Traceback (most recent call last):\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/urllib3/connection.py\", line 174, in _new_conn\n    conn = connection.create_connection(\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/urllib3/util/connection.py\", line 95, in create_connection\n    raise err\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/urllib3/util/connection.py\", line 85, in create_connection\n    sock.connect(sa)\nsocket.timeout: timed out\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/urllib3/connectionpool.py\", line 703, in urlopen\n    httplib_response = self._make_request(\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/urllib3/connectionpool.py\", line 386, in _make_request\n    self._validate_conn(conn)\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/urllib3/connectionpool.py\", line 1042, in _validate_conn\n    conn.connect()\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/urllib3/connection.py\", line 363, in connect\n    self.sock = conn = self._new_conn()\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/urllib3/connection.py\", line 179, in _new_conn\n    raise ConnectTimeoutError(\nurllib3.exceptions.ConnectTimeoutError: (<urllib3.connection.HTTPSConnection object at 0x7fbfcb4f6460>, 'Connection to api.github.com timed out. (connect timeout=4)')\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/requests/adapters.py\", line 486, in send\n    resp = conn.urlopen(\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/urllib3/connectionpool.py\", line 787, in urlopen\n    retries = retries.increment(\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/urllib3/util/retry.py\", line 592, in increment\n    raise MaxRetryError(_pool, url, error or ResponseError(cause))\nurllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.github.com', port=443): Max retries exceeded with url: /repos/conda-forge/mysql-connector-python-feedstock (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fbfcb4f6460>, 'Connection to api.github.com timed out. (connect timeout=4)'))\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/github3/models.py\", line 201, in _request\n    return request_method(*args, **kwargs)\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/requests/sessions.py\", line 602, in get\n    return self.request(\"GET\", url, **kwargs)\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/github3/session.py\", line 171, in request\n    response = super().request(*args, **kwargs)\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/requests/sessions.py\", line 589, in request\n    resp = self.send(prep, **send_kwargs)\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/requests/sessions.py\", line 703, in send\n    r = adapter.send(request, **kwargs)\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/requests/adapters.py\", line 507, in send\n    raise ConnectTimeout(e, request=request)\nrequests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.github.com', port=443): Max retries exceeded with url: /repos/conda-forge/mysql-connector-python-feedstock (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fbfcb4f6460>, 'Connection to api.github.com timed out. (connect timeout=4)'))\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/auto_tick.py\", line 1187, in _run_migrator\n    migrator_uid, pr_json = run(\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/auto_tick.py\", line 217, in run\n    feedstock_dir, repo = get_repo(\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/git_utils.py\", line 286, in get_repo\n    repo = gh.repository(\"conda-forge\", feedstock_reponame)\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/github3/github.py\", line 2103, in repository\n    json = self._json(self._get(url), 200)\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/github3/models.py\", line 216, in _get\n    return self._request(\"get\", url, **kwargs)\n  File \"/home/runner/micromamba/envs/cf-scripts/lib/python3.9/site-packages/github3/models.py\", line 206, in _request\n    raise exceptions.ConnectionError(exc)\ngithub3.exceptions.ConnectionError: <class 'requests.exceptions.ConnectTimeout'>: A connection-level exception occurred: HTTPSConnectionPool(host='api.github.com', port=443): Max retries exceeded with url: /repos/conda-forge/mysql-connector-python-feedstock (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fbfcb4f6460>, 'Connection to api.github.com timed out. (connect timeout=4)'))\n",
  "pypy37": "not solvable: master: ['linux_aarch64_python3.7.____73_pypy: Encountered problems while solving.\\nProblem: package protobuf-3.11.4-py36h831f99a_1 requires python_abi 3.6.* *_cp36m, but none of the providers can be installed\\n']"
 },
 "smithy_version": "3.23.0"
}